parentchildassignment/src/components/atoms/Button/Button.tsx

```diff
@@ -0,0 +1,38 @@
+import { ThemeProvider } from "@mui/material";
+import Button from "@mui/material/Button";
+import React from "react";
+import theme from "../../../theme";
+
+
+const CustomButton: React.FC = () => {
+  return (
+    <ThemeProvider theme={theme}>
+      <div style={{ display: "flex", flexDirection: "column" }}>
```

Comment:
https://github.com/YaminiNallakunta/yamininallakunta_ldp/pull/3/files#r1678863848
Also try to use <Box></Box> instead of div.